app/bundles/CoreBundle/Config/config.php

```diff
@@ -1187,7 +1190,7 @@
         'db_table_prefix'                 => '',
         'db_server_version'               => '5.7',
         'locale'                          => 'en_US',
-        'secret_key'                      => '',
+        'secret_key'                      => 'temp',
```

Comment:
Note to self: needed this to get the installer to work (since no `secret_key` is set at that point yet). Might have to find a different solution here; discuss in #mautic-4 Slack channel later

Reply:
Have added it to the Mautic 4 JIRA board for looking into it later: https://mautic.atlassian.net/browse/TPROD-169